recipesesame/src/main/java/com/recipesesame/functions/Recipe.java

```diff
@@ -95,7 +95,24 @@ public void setCookTime(int cookTime) {
 	public void setInstructions(ArrayList<Step> instructions) {
 		this.instructions = instructions;
 	}
+	public void displayAll(){
+		System.out.println(this.title);
+		System.out.println();
+		System.out.println(this.subtitle);
+		System.out.println();
+		System.out.println("Serves: "+this.servingSize+"		Prep Time: "+this.prepTime+"		Cook Time: "+this.cookTime);
+		System.out.println();
+		System.out.println("Ingredients:");
+		for(int i=0;, i<this.ingredients.size(); i++){
+			System.out.println(this.ingredients.get(i).toString()+',');
+		}
+		System.out.println();
+		System.out.println("Instructions: ");
+		for(int i=0; i<this.instructions.size(); i++){
+			System.out.println(this.instructions.get(i).toString()+',');
```

Comment:
Do we need a comma after each instruction? We should also consider numbering each instruction.